resources/migrations/000_migrations.yaml

```diff
@@ -15350,6 +15350,92 @@ databaseChangeLog:
                   type: varchar(100)
                   remarks: 'Metabase version used to create the card.'
 
+  - changeSet:
+      id: v48.00-022
+      author: johnswanson
+      comment: Migrate migrate-click-through to a custom migration
+      preConditions:
+        - onFail: MARK_RAN
+        - sqlCheck:
+            expectedResult: 0
+            sql: SELECT count(*) FROM data_migrations WHERE id = 'migrate-click-through';
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.MigrateClickThrough"
+
+  - changeSet:
+      id: v48.00-023
+      author: piranha
+      comment: Data migration migrate-remove-admin-from-group-mapping-if-needed
+      preConditions:
+        - onFail: MARK_RAN
+        - sqlCheck:
+            expectedResult: 0
+            sql: SELECT count(*) FROM data_migrations WHERE id = 'migrate-remove-admin-from-group-mapping-if-needed';
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.MigrateRemoveAdminFromGroupMappingIfNeeded"
+
+  - changeSet:
```

Comment:
btw, I'm thinking about adding to the lint check to make sure only one change is allowed in the rollback too.

Do you think you can break it into 2 here? also maybe we need to check with BE core to see what's the reason behind the rule of one change set per migration only.

Reply:
Okay let me ask then about this rule first and then let's decide on what to do with splitting. I'm not a fan of splitting just because this feels like it should be a single change rather than two distinct changes, but let me first find out the reasons behind the rule. :)

Reply:
Okay so I've [read liquibase best practices](https://www.liquibase.org/get-started/best-practices#tabbing-block_6168671689f27) about a single change in a changeset and I'm not convinced it's a good idea to split this rollback into two transactions... 

I mean technically we could, but logically 1 change drops table, the other one returns it back, sounds like a good way to structure changes?

Reply:
yea well, it's good practice, not rules so I think as long as we don't do anything crazy it's fine.

but I have an idea if you really want to have just one change you can turn the create table to SQL and have it in one single change set with the insert.

That's way the lint don't have to handle  exceptions